tasks/main.yml

```diff
@@ -86,10 +86,12 @@
   template:
     src: proot.conf.j2
     dest: "{{ dpb_proot_conf_file }}"
+  notify:
+    - Update chroot
 
 - name: Create chroot
```

Comment:
it seems this task is not necessary as the task "Create proot.conf" notifies "Update chroot" and the chroot will be created if not exists.